HIP/hip-1249.md

```diff
@@ -153,6 +153,11 @@ operations not covered by other throttles.
 
 If a precompile/system call exhausts ops mid-transaction, handle as in Execution Outcomes.
 
+## Mirror Node Considerations
```

Comment:
There can be an empty line below the heading, so that it's better separated as the rest of the paragraphs.